<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObavezaQt</class>
 <widget class="QWidget" name="ObavezaQt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>276</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Tip</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="TipEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Broj mogucih poena</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="MoguciEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Broj osvojenih poena</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="OsvojeniEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Vrijeme</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="VrijemeEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Rok</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="RokEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Komentar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="KomentarEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="Dodaj">
      <property name="text">
       <string>Dodaj</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Dodaj</sender>
   <signal>clicked()</signal>
   <receiver>ObavezaQt</receiver>
   <slot>dodaj()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>dodaj()</slot>
 </slots>
</ui>
